### ISSUE TITLE
Mitie.py in python 2.7 binary does not match Mitie.py master

### ISSUE THREAD

**Issue by shailesh030** [mit-nlp/MITIE#154]
When trying to refer to mitie.total_word_feature_extractor by importing mitie.py from latest python 2.7 binary, getting an error  

AttributeError: 'module' object has no attribute 'total_word_feature_extract

mitie.py in binary version is missing the function total_word_feature_extract. Looks like the binary needs to be refreshed to reflect changes in master. Is it possible to upload the latest python binary?

**Comment by davisking** [comment]
Yeah, it's been a while since I uploaded new precompiled binaries.  It's probably about time to refresh them :)

**Comment by modadugu** [comment]
Can you please share the link of uploaded binaries. I am not seeing any updates in the download section

**Comment by davisking** [comment]
I haven't updated anything yet.  Be patient.
